.vscode/settings.json

```diff
@@ -6,5 +6,9 @@
 {
     "tslint.enable": true,
     "tslint.configFile": "./config/tslint/tslint.json",
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "search.exclude": {
+        "**/node_modules": true,
```

Comment:
i am not sure about `node_modules` i look up some typings periodically from there... we can give it a try